README.md

```diff
@@ -1,9 +1,9 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-My Personal Portfolio(Mobile)-Microverse
+My Personal Portfolio(Desktop)-Microverse
 
 > My Portfolio
-This is a my first phase project on my personal portfolio development with microverse created using simple HTML(Hyper text mark-up Language.) and CSS(Cascading Style Sheet) which is used for styling or decorating the webpage.
+This is a my first phase project on my personal portfolio development with microverse created using simple HTML(Hyper text mark-up Language.) and CSS(Cascading Style Sheet) which is used for styling or decorating the webpage. It is the desktop version.
 
```

Comment:
You can put a link to github pages then you can show your work inmediately to the word.